core/appeal/service_test.go

```diff
@@ -841,7 +839,7 @@ func (s *ServiceTestSuite) TestCreate() {
 			s.mockNotifier.On("Notify", mock.Anything).Return(nil).Once()
 			s.mockAuditLogger.On("Log", mock.Anything, appeal.AuditKeyBulkInsert, mock.Anything).Return(nil).Once()
 
-			err := s.service.Create(ctx, []*domain.Appeal{input})
+			err := s.service.Create(context.TODO(), []*domain.Appeal{input}, appeal.WithAdditionalAppealOption())
```

Comment:
can we use `context.Background()` instead here? since this call is the top level execution in this case